src/Map/index.js

```diff
@@ -70,6 +74,15 @@ export default class Map extends Component {
     }
   }
 
+  async componentDidUpdate(prevPops) {
+    const { markerCollection, editor } = prevPops
+    const { loaded } = this.state
+
+    if (markerCollection && !loaded && !editor) {
+      this.loadAddresses(prevPops)
```

Comment:
Should this get called on first render as well? I.e., in `componentDidMount()`

Reply:
No, because `markerCollection` is always falsy on-mount. I can leave a comment to that effect.